mkdocs.yml

```diff
@@ -17,7 +17,7 @@ markdown_extensions:
   - pymdownx.tilde
 site_favicon: 'images/favicon.ico'
 repo_name: 'jaegertracing/jaeger'
-repo_url: 'https://github.com/jaegertracing/jaeger'
+repo_url: 'https://github.com/jaegertracing/documentation'
```

Comment:
sorry, we don't want to change this, because it makes the top-right corner populated with details about the docs repo instead of the main repo.

did you find some issues caused by this link?